components/EntityAttributeValue.js

```diff
@@ -43,7 +43,14 @@ const formatImage = (key) => {
       case 'custom':
         const CustomAttributeType = plugin(item.attributes_custom_name);
         const customAttributeType = new CustomAttributeType({
-          data: item.value_long_string, metadata: {type: 'custom', custom_name: item.attributes_custom_name}
+          data: {
+            value: item.value_long_string, 
+            id: item.id
+          },
+          metadata: {
+            type: 'custom',
+            custom_name: item.attributes_custom_name
+          }
```

Comment:
This will be incompatible with existing AttributeTypes, where we expect data to be data "as-is", without the wrapping of an object. What I think we can do is put the `id` in the metadata instead.

```suggestion
          data: item.value_long_string, 
          metadata: {
            type: 'custom',
            custom_name: item.attributes_custom_name,
            id: item.id
          }
```